src/driver.mli

```diff
@@ -81,8 +81,11 @@ end
     [lint_impl] and [lint_intf] are applied to the unprocessed source. Errors they return
     will be reported to the user as preprocessor warnings.
 
-    [preprocess_impl] and [preprocess_intf] are applied after linters,
-    but before other transformations.
+    Rewritings are applied in the following order:
+    - linters ([lint_impl], [lint_intf])
+    - preprocessing ([preprocess_impl], [preprocess_intf])
+    - contest-independent rules ([rules], [extensions])
```

Comment:
Context*?

Reply:
fixed

Reply:
Uh, it was merged before I "fixed" it. I'll make another PR.